jwala-services/src/main/java/com/cerner/jwala/service/jvm/impl/JvmServiceImpl.java

```diff
@@ -727,9 +727,9 @@ protected void deployScriptsToUserJwalaScriptsDir(Jvm jvm, User user) throws Com
 	/**
 	 * This method upgrades the JDK version for the mentioned JVM

```

Comment:
Add more to this description here stating that the JVM is expected to be in a STOPPED state, and that the JVM is expected to be configured with a setenv.bat/sh file in order to update the JAVA_HOME.